pulp/tests/test_pulp.py

```diff
@@ -648,7 +649,7 @@ def test_pulp_123(self):
         elif self.solver.__class__ in [CPLEX_DLL, GUROBI_CMD]:
             # GLPK_CMD Does not report unbounded problems, correctly
             pulpTestCheck(prob, self.solver, [const.LpStatusNotSolved])
-        elif self.solver.__class__ in [PULP_CHOCO_CMD, CHOCO_CMD]:
+        elif self.solver.__class__ in [PULP_CHOCO_CMD, CHOCO_CMD, SCIP_CMD]:
```

Comment:
here too, can we match the actual status from SCIP from one of the above if clauses?

Reply:
Sorry I think i misread this and the other one, they both should have been in the group above, categorised now.